examples/misc_instancing.html

```diff
@@ -0,0 +1,231 @@
+<html>
+    <head>
+        <title>Itowns - Instancing</title>
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" type="text/css" href="css/example.css" />
+        <link rel="stylesheet" type="text/css" href="css/LoadingScreen.css" />
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.7.6/dat.gui.min.js"></script>
+    </head>
+
+    <body style="background-color: black">
```

Comment:
Why setting this style ? If this is needed, could you move it in css file ?

Reply:
Not needed you're right